keyboards/clawsome/gamebuddy/v1_M.M/config.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2021 AAClawson (AlisGraveNil)
```

Comment:
This folder still needs to be renamed.

Reply:
I can't get it to rename - I got it to list "v1_m.M" but I can't get rid of the ".M" part of the folder name

Reply:
Try using `git mv`.